### DISCUSSION TITLE
A lot of examples to inspire your own wiki.js

### DISCUSSION THREAD

**Discussion by fguisso** [requarks/wiki#6584] [Showcase your Wiki]
I trying to figure out how to organize my wiki.js homepage and it's not a very easy task to be done, then I'm sharing my research, using Google Dorks and Wappalizer to list some servers using wiki.js:

- https://docs2.hubitat.com/
- https://guide.switchupcb.com/
- https://bromberglab.org
- https://wiki-sagi.cidadania.gov.br/
- https://zoonosis.kemkes.go.id/
- https://support.endlessos.org/en/home
- https://redteam.wiki/
- https://wiki.cova-cci.org/
- https://wiki.bambulab.com/en/home
- https://www.alswiki.org/en/home
- https://wiki.bambulab.com/en/home
- https://www.alswiki.org/en/home
- https://wiki.ivao.aero/en/home

**Comment by giuliolibrando** [comment]
I would also add the BambuLab wiki to this discussion https://wiki.bambulab.com/en/home

**Comment by jj-sm** [comment]
I would also add [wiki.ivao.aero](wiki.ivao.aero) ✈️

**Comment by HelioPri** [comment]
has anybody got an example with AI integrated search?

**Comment by biscottiman** [comment]
Playergroup in Elite Dangerous: [https://wiki.antixenoinitiative.com/en/home](https://wiki.antixenoinitiative.com/en/home)

**Comment by ckozlov-ASR** [comment]
This post inspired me to build some CSS color theme templates.

https://github.com/ckozlov-ASR/wikijs_css_themes
